src/platform/platform_posix.c

```diff
@@ -519,11 +524,14 @@ CxPlatGetAbsoluteTime(
 #if defined(CX_PLATFORM_LINUX)
     ErrorCode = clock_gettime(CLOCK_MONOTONIC, Time);
 #elif defined(CX_PLATFORM_DARWIN)
-    timespec_get(Time, TIME_UTC);
+    //
+    // timespec_get is used on darwin, as CLOCK_MONOTONIC isn't actually
+    // monotonic according to our tests.
+    //
+    ErrorCode = timespec_get(Time, TIME_UTC);
 #endif // CX_PLATFORM_DARWIN
 
-    CXPLAT_DBG_ASSERT(ErrorCode == 0);
-    UNREFERENCED_PARAMETER(ErrorCode);
+    CXPLAT_DBG_ASSERT(ErrorCode != 0);
```

Comment:
You changed this assert to be the opposite of what it was? Is that not going to break Linux?

Reply:
Realized that so just fixed it.